pkg/k8shandler/util.go

```diff
@@ -19,10 +19,10 @@ const (
 	LinuxValue  = "linux"
 )
 
-// ensureLinuxNodeSelector takes given selector map and returns a selector map with linux node selector added into it.
+// EnsureLinuxNodeSelector takes given selector map and returns a selector map with linux node selector added into it.
 // If there is already a node type selector and is different from "linux" then it is overridden and warning is logged.
 // See https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#interlude-built-in-node-labels
-func ensureLinuxNodeSelector(selectors map[string]string) map[string]string {
+func EnsureLinuxNodeSelector(selectors map[string]string) map[string]string {
```

Comment:
reason for case/scope change?

Reply:
https://github.com/openshift/elasticsearch-operator/pull/211/files/e9fb148eef5b250160e751d649c0c7051c67e0ee#r363899713

Reply:
right, but nothing outside of the `k8shandler` package is calling that currently. If this is in part of an effort to break up that package shouldn't we wait for that time to change this?

Reply:
I will revert for this PR as I don't see it here.